internal/lookout/ui/src/services/ComputeResourcesService.ts

```diff
@@ -0,0 +1,89 @@
+import { load } from "js-yaml"
+
+// A simple interface that our PodSpec has.
+interface JobYaml {
+  podSpec: {
+    containers: Container[]
+  }
+}
+interface Container {
+  command: string[]
+  args: string[]
+  resources: {
+    limits: Limits
+  }
+}
+
+interface Limits {
+  cpu?: string
+  "nvidia.com/gpu"?: string
+  storage?: string
+  memory?: string
+}
+
+export function convertStringToYaml(jobYaml: string): JobYaml {
+  return load(jobYaml) as JobYaml
+}
+
+function checkContainerFieldExists(yamlifed: JobYaml): boolean {
+  return !yamlifed.podSpec.containers
+}
+
+export function getCommandFromJobYaml(yamlifed: JobYaml): string[] | undefined {
+  if (checkContainerFieldExists(yamlifed)) return
+  const stringArray: string[] = []
+  yamlifed.podSpec.containers.map((val) => {
+    if (val.command) {
+      stringArray.push(val.command.join(" "))
+    } else {
+      stringArray.push("")
+    }
+  })
+  return stringArray
+}
+export function getCommandArgumentsFromJobYaml(yamlifed: JobYaml): string[] | undefined {
+  if (checkContainerFieldExists(yamlifed)) return
+  const stringArray: string[] = []
+  yamlifed.podSpec.containers.map((val) => {
+    if (val.args) {
+      stringArray.push(val.args.join(" "))
+    } else {
+      stringArray.push("")
+    }
+  })
+  return stringArray
+}
+
+export function getCpuFromJobYaml(yamlifed: JobYaml): string[] | undefined {
```

Comment:
```suggestion
export function getCpuFromJobYaml(yamlified: JobYaml): string[] | undefined {
```
Likewise for the others